docs/versions/latest/Release.md

```diff
@@ -1,34 +1,37 @@
-[//]: # (VERSION_PLACEHOLDER DO NOT DELETE)
-[//]: # (Used when working on a new release. Placed together with the Version.md)
-[//]: # (Nothing here is optional. If a step must not be performed, it must be said so)
-[//]: # (Do not fill the version, it will be done automatically)
-[//]: # (Quick Intro to what is the focus of this release)
+[//]: # "VERSION_PLACEHOLDER DO NOT DELETE"
```

Comment:
Lets revert these changes

Reply:
Discussed with @VenelinBakalov , no need for reverting currently.

Reply:
There is no urgency to merge, let's wait for @ktsonchev to come back and we can cover this with him to make sure everyone is alligned.